crates/phactory/src/prpc_service.rs

```diff
@@ -1145,6 +1134,9 @@ impl<Platform: pal::Platform + Serialize + DeserializeOwned> PhactoryApi for Rpc
                         .map_err(|_| from_display("Invalid received RA report"))?;
                     ias_fields.extend_mrenclave()
                 }
+                _ => {
```

Comment:
Match arms could cover all variants for better maintainability.

Reply:
Changed, the None attestation report shouldn't happen so I panic on the branch